models/system/databricks-dolly-v2-12b/description.md

```diff
@@ -0,0 +1,25 @@
+The dolly-v2-12b model is a large, instruction-following language model created by Databricks, Inc. and is licensed for commercial use. It is based on EleutherAI’s Pythia-12b and was trained on a 15,000-record instruction corpus generated by Databricks employees, which was released under a CC-BY-SA license. It is not a state-of-the-art model, but it does exhibit high-quality instruction-following behavior that is not characteristic of the foundation model on which it is based. The model is available in three sizes: dolly-v2-12b, dolly-v2-7b, and dolly-v2-3b.
+
+> The above summary was generated using ChatGPT. Review the <a href="https://huggingface.co/databricks/dolly-v2-12b" target="_blank">original model card</a> to understand the data used to train the model, evaluation metrics, license, intended uses, limitations and bias before using the model.
+
+### Sample inputs and outputs (for real-time inference)
+
+```json
+{
+    "inputs": {
+        "input_string": ["My name is John and I am", "Once upon a time,"]
```

Comment:
@haritha-bendapudi please update this to working sample since we are using pyfunc flavor. 

Cc: @praveenmathamsetty

Reply:
Done